### ISSUE TITLE
Retrieve images over ssl

### ISSUE THREAD

**Issue by katyedwards** [healthlocker/healthlocker#1172]
Images for stories re not currently retrieved over ssl meaning users get a 'not fully secure' alert when they come to the site.

Please can we have a v quick time estimate on whether this is something we can possibly address,  and if it is, approx ballpark time estimate.

**Comment by RobStallion** [comment]
![image](https://user-images.githubusercontent.com/15571853/37704238-3a16027e-2cf0-11e8-9c0d-792078a6ab75.png)

This image is pulled from a site that has https (as shown in the link below) and we can see that the site is still fully secure.
```
style="background-image:url(https://b-i.forbesimg.com/glassheel/files/2013/05/15814393_m.jpg)"
```

However...

![image](https://user-images.githubusercontent.com/15571853/37704420-98f66f90-2cf0-11e8-8e69-7b3b7ddd57a0.png)

this image is sourced from a `http` site and we now get the warning
```
style="background-image:url(http://www.slam.nhs.uk/media/79506/debbie_300x200.jpg)"
```

I would suggest that the admin makes sure to source images from a secure site.

**Comment by katyedwards** [comment]
Thanks @RobStallion any idea whether this is something we could fix technically?

The images on the slam website are the actual person whose story it is, so they may not feel it appropriate to display different images.

Just a ballpark estimate if it is possible so I have all the info to discuss with client.   Thanks!

**Comment by RobStallion** [comment]
@katyedwards I do not think that this is an issue on our end. The issue is with where the images are being sourced which is why some of them are ok.

My first suggestion would be to make sure that they source them from a secure site. If for some reason they cannot do this then we would have to rethink the way an admin adds stories. 

The only way to keep the ssl happy would be to source the images from the repo itself, or to source them from a `https` site. If slam's site is not `https` then we would potentially have to set up something like `amazon s3` to store the images on and then source them from here
